mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java

```diff
@@ -127,4 +143,20 @@ void showToast(String message) {
     void generateQR(String qrData) {
         mShowQrPresenter.generateQr(qrData);
     }
+
+    @Override
+    public void showMobile(String mobileNo) {
+        PhoneNumberUtil phoneNumberUtil =
```

Comment:
No need to add a new library altogether, we can use a simple regex for parsing phone number 
`"^(\\+91)*0*([987]\\d{9})\$"`

Reply:
Could you explain further? We need a function to show the phone number  Are you trying to way we have to remove the function as well?

Reply:
I am hesitant to include a new library just for this usecase `io.michaelrocks:libphonenumber-android:8.11.2`
We can display the phone number as is without formatting. Its a fairly big library last time i checked

Reply:
Sure, let me check how it's done

Reply:
I guess we already have that library for some other purpose. I shall take care of the formatting and create a new pr as I have re formatted my system and all the stuff is gone

Reply:
@Bhavnaharitsa update on this?

Reply:
Yess.. like I have mentioned before, we are already using the library. I am not introducing it in the issue. So I guess we can keep it. What are your thoughts? @luckyman20

Reply:
Well, I see that you are putting it in `build.gradle` file which means that you are introducing it otherwise if it was being used in app then it would have been present there.

Reply:
@luckyman20 Yes, you're right. But unfortunately, I am forced to close this pr as I have lost my branches during this time of contribution due to system format. Please bare with the multiple prs. 
But, I just checked we already have this dependency. I shall make sure i dont add it in my next pr

Reply:
Okay, no issues. Also, make sure all other comments are also incorporated in the new PR.

Reply:
Sure, will make note of it